create-publication-from-scopus/src/main/java/no/sikt/nva/scopus/conversion/CristinContributorExtractor.java

```diff
@@ -0,0 +1,103 @@
+package no.sikt.nva.scopus.conversion;
+
+import static java.util.Objects.nonNull;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+import no.scopus.generated.AuthorTp;
+import no.scopus.generated.PersonalnameType;
+import no.sikt.nva.scopus.conversion.model.cristin.Affiliation;
+import no.sikt.nva.scopus.conversion.model.cristin.Person;
+import no.sikt.nva.scopus.conversion.model.cristin.TypedValue;
+import no.unit.nva.model.Contributor;
+import no.unit.nva.model.Identity;
+import no.unit.nva.model.Organization;
+import no.unit.nva.model.Role;
+import nva.commons.core.JacocoGenerated;
+import nva.commons.core.StringUtils;
+import org.jetbrains.annotations.NotNull;
+
+public class CristinContributorExtractor {
+
+    public static final String NAME_DELIMITER = ", ";
+    public static final String FIRST_NAME_CRISTIN_FIELD_NAME = "FirstName";
+    public static final String LAST_NAME_CRISTIN_FIELD_NAME = "LastName";
+    public static final String ORCID_FIELD_NAME = "orcid";
+
+    @JacocoGenerated
+    public CristinContributorExtractor() {
```

Comment:
En public constructor gjør det mulig å instansiere klassen. God skikk å gjøre default constructor private for utility-klasser med bare static metoder.

Reply:
fikset